servus/serializable.h

```diff
@@ -99,23 +100,28 @@ class Serializable
 #endif
 
     /**
-     * Set a new function called after the object has been updated.
-     *
-     * @return the previously set function.
+     * Set a new function called after every change on the local object
+     * instance (e.g. Collage distributable).
      */
-    SERVUS_API ChangeFunc setUpdatedFunction( const ChangeFunc& func );
+    SERVUS_API void setLocalChangeFunction( const ChangeFunc& func );
+
+    /**
+     * Set a new function called after the object has been updated remotely
+     * (via a subscriber, a http server, loading from file...).
+     */
+    SERVUS_API void setRemoteUpdateFunction( const ChangeFunc& func );
```

Comment:
True, a mechanism to unset is needed. To keep the API simple is it possible to get away with setting a "null" function instead of having an explicit unset? (This only works in the case of a single callback of course.)